src/generators/Minecraft.res

```diff
@@ -117,6 +118,25 @@ module Cuboid = {
       rotate: 0.0,
     }
 
+    // When a face is flipped both vertically and horizontally, this is the same as rotating 180 degrees.
```

Comment:
One more thought, I'm guessing for our use cases we won't need to apply multiple flips? In that case you might even like to simplify this further to:

```
let setFlip = (face: t, flip: Generator_Texture.flip) => {
  ...face,
  flip
}
```

Something in the back of my mind is Generator V2 where everything is based on 2D coordinates, which is a bit more conventional for having a "pipeline" of multiple operations like that.

Reply:
For our use cases, occasionally we have a 180 degree rotate from the direction step that needs to be addressed by the center step. I've also thought about how since these flips, and rotates in the case of things such as the old character model's left arm and leg, are moreso about how the source texture is mapped than being exclusive to the destination, so it may make more sense to have rotations and flips like those included in the source step.